apitests/spec/devices_spec.js

```diff
@@ -0,0 +1,34 @@
+const request = require('request').defaults({ json: true, baseUrl: 'https://localhost:44305/api/v1/devices' });
```

Comment:
so are we planning to push these tests to the final RM v1.5 version? Or is it just for our convenience?